libs/designer-ui/src/lib/dictionary/util/serializecollapseddictionary.ts

```diff
@@ -16,7 +16,7 @@ export const serializeDictionary = (
 ) => {
   editor.getEditorState().read(() => {
     const nodeMap = new Map<string, ValueSegment>();
-    const editorString = getChildrenNodes($getRoot(), nodeMap);
+    const editorString = getChildrenNodesWithTokenInterpolation($getRoot(), nodeMap);
```

Comment:
FYI, doing this, if a user edits their dictionary in collapsedView, it will serialize with the "" around the token still